### PR TITLE
Add kirenenko

### PR DESCRIPTION
**Pull request by chenju2k6**
Signed-off-by: Chen Ju <chenju2k6@gmail.com>

### CODE REVIEW

**Comment by chenju2k6** [comment]
> Thanks for your pull request. It looks like this may be your first contribution to a Google open source project (if not, look below for help). Before we can look at your pull request, you'll need to sign a Contributor License Agreement (CLA).
> 
> 📝 **Please visit https://cla.developers.google.com/ to sign.**
> 
> Once you've signed (or fixed any issues), please reply here with `@googlebot I signed it!` and we'll verify it.
> 
> #### What to do if you already signed the CLA
> ##### Individual signers
> * It's possible we don't have your GitHub username or you're using a different email address on your commit. Check [your existing CLA data](https://cla.developers.google.com/clas) and verify that your [email is set on your git commits](https://help.github.com/articles/setting-your-email-in-git/).
> 
> ##### Corporate signers
> * Your company has a Point of Contact who decides which employees are authorized to participate. Ask your POC to be added to the group of authorized contributors. If you don't know who your Point of Contact is, direct the Google project maintainer to [go/cla#troubleshoot](http://go/cla#troubleshoot) ([Public version](https://opensource.google/docs/cla/#troubleshoot)).
> * The email used to register you as an authorized contributor must be the email used for the Git commit. Check [your existing CLA data](https://cla.developers.google.com/clas) and verify that your [email is set on your git commits](https://help.github.com/articles/setting-your-email-in-git/).
> * The email used to register you as an authorized contributor must also be [attached to your GitHub account](https://github.com/settings/emails).
> 
> ℹ️ **Googlers: [Go here](https://goto.google.com/prinfo/https%3A%2F%2Fgithub.com%2Fgoogle%2Ffuzzbench%2Fpull%2F883) for more info**.

**Comment by chenju2k6** [comment]
> Thanks for your pull request. It looks like this may be your first contribution to a Google open source project (if not, look below for help). Before we can look at your pull request, you'll need to sign a Contributor License Agreement (CLA).
> 
> 📝 **Please visit https://cla.developers.google.com/ to sign.**
> 
> Once you've signed (or fixed any issues), please reply here with `@googlebot I signed it!` and we'll verify it.
> 
> #### What to do if you already signed the CLA
> ##### Individual signers
> * It's possible we don't have your GitHub username or you're using a different email address on your commit. Check [your existing CLA data](https://cla.developers.google.com/clas) and verify that your [email is set on your git commits](https://help.github.com/articles/setting-your-email-in-git/).
> 
> ##### Corporate signers
> * Your company has a Point of Contact who decides which employees are authorized to participate. Ask your POC to be added to the group of authorized contributors. If you don't know who your Point of Contact is, direct the Google project maintainer to [go/cla#troubleshoot](http://go/cla#troubleshoot) ([Public version](https://opensource.google/docs/cla/#troubleshoot)).
> * The email used to register you as an authorized contributor must be the email used for the Git commit. Check [your existing CLA data](https://cla.developers.google.com/clas) and verify that your [email is set on your git commits](https://help.github.com/articles/setting-your-email-in-git/).
> * The email used to register you as an authorized contributor must also be [attached to your GitHub account](https://github.com/settings/emails).
> 
> ℹ️ **Googlers: [Go here](https://goto.google.com/prinfo/https%3A%2F%2Fgithub.com%2Fgoogle%2Ffuzzbench%2Fpull%2F883) for more info**.

@googlebot I signed it!

**Comment by inferno-chromium** [comment]
Looks like you accidently closed it, reopening.

**Comment by inferno-chromium** [comment]
@lszekeres - can you please help with round 1 review of this.

File: fuzzers/kirenenko/runner.Dockerfile [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+COPY fuzz_dtlsclient_kir /out/fuzz_dtlsclient_kir
+COPY fuzz_dtlsclient_vani /out/fuzz_dtlsclient_vani
+COPY decode_fuzzer_kir /out/decode_fuzzer_kir
+COPY decode_fuzzer_vani /out/decode_fuzzer_vani
+COPY ossfuzz_kir /out/ossfuzz_kir
+COPY ossfuzz_vani /out/ossfuzz_vani
+COPY libpng_read_fuzzer_kir /out/libpng_read_fuzzer_kir
+COPY libpng_read_fuzzer_vani /out/libpng_read_fuzzer_vani
+COPY libjpeg_turbo_fuzzer_kir /out/libjpeg_turbo_fuzzer_kir
+COPY libjpeg_turbo_fuzzer_vani /out/libjpeg_turbo_fuzzer_vani
```
Comment (lszekeres) [inline fuzzers/kirenenko/runner.Dockerfile:66]: As discussed offline, let's remove all these pre-compiled binaries first. Thanks!

**Review by lszekeres — CHANGES_REQUESTED** [review]
Thank you for integrating Kirenenko! Very exciting!

File: fuzzers/kirenenko/runner.Dockerfile (hunk 1 above)
Comment (chenju2k6) [reply fuzzers/kirenenko/runner.Dockerfile:66]: Can I keep aflgrader and afl-qemu-trace, those are used in the hybrid fuzzing. They are also compiled from a private repo.

Comment (lszekeres) [reply fuzzers/kirenenko/runner.Dockerfile:66]: Sorry, we cannot check in and run pre-compiled binaries in fuzzbench. Those tools should be built together with afl in `builder.Dockerfile`.

Comment (chenju2k6) [reply fuzzers/kirenenko/runner.Dockerfile:66]: Got it. Thanks!

Comment (chenju2k6) [reply fuzzers/kirenenko/runner.Dockerfile:66]: I opened a new PR which removes all the binaries. I am gonna close this one. Thanks!

**Pull request closed without merging**